homeassistant/components/atag/.translations/en.json

```
@@ -0,0 +1,20 @@
+{
+  "config": {
+    "title": "Atag",
```

```suggestion
    "title": "Atag",
    "config": {
```

title is now top level

---

Caught that one during the pre-commit :)